services/blockchain-indexer/shared/indexer/transactionProcessor/pos/registerValidator.js

```diff
@@ -30,24 +30,41 @@ const MYSQL_ENDPOINT = config.endpoints.mysql;
 const accountsTableSchema = require('../../../database/schema/accounts');
 const transactionsTableSchema = require('../../../database/schema/transactions');
 const validatorsTableSchema = require('../../../database/schema/validators');
+const commissionsTableSchema = require('../../../database/schema/commissions');
 
 const { getPosConstants } = require('../../../dataService');
 const { requestConnector } = require('../../../utils/request');
 
 const getAccountsTable = () => getTableInstance(accountsTableSchema, MYSQL_ENDPOINT);
 const getTransactionsTable = () => getTableInstance(transactionsTableSchema, MYSQL_ENDPOINT);
 const getValidatorsTable = () => getTableInstance(validatorsTableSchema, MYSQL_ENDPOINT);
+const getCommissionsTable = () => getTableInstance(commissionsTableSchema, MYSQL_ENDPOINT);
 
 // Command specific constants
 const COMMAND_NAME = 'registerValidator';
 
+const getCommissionIndexingInfo = async (blockHeader, tx) => {
+	// TODO: Enable this to fetch default commission value from node once SDK fixes https://github.com/LiskHQ/lisk-sdk/issues/8856
+	// const posConstants = await getPosConstants();
+	// const defaultComission = posConstants.defaultCommission;
+
+	const newCommissionEntry = {
+		address: getLisk32AddressFromPublicKey(tx.senderPublicKey),
```

Comment:
I think, we can avoid this extra function call because `senderAddress` is already available in the transaction. We are resolving this in `blockchainIndex.js` file

```suggestion
		address: tx.senderAddress,
```